lte/gateway/c/oai/lib/directoryd/directoryd.h

```diff
@@ -36,6 +36,8 @@ bool directoryd_remove_location(char* imsi);
 
 bool directoryd_update_location(char* imsi, char* location);
 
+bool directoryd_update_field(char* imsi, char* key, char* value);
```

Comment:
should this be named `directoryd_update_record_field`? seems more aligned with what the function does and with the function name on GatewayDirectorydClient